pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -120,6 +136,17 @@
 
   .clear{style: "clear: both"}
 
+- if show_census_announcement
+  .special_announcement_mobile.mobile-feature{style: "background-color: #00adbc"}
+    .bigannounce{style: "padding: 20px; text-align: center; color: white"}
+      .bigheading{style: "font-size: 30px; line-height: 1.2em"}
+        = census_announcement['homepage_text']
+      .subheading{style: "padding-top: 20px"}
+        = census_announcement['homepage_subtext']
+      %a{href: "/yourschool"}
+        %button{style: "margin-top: 20px"}
+          Learn more
```

Comment:
need to be localized? i suspect we already have this string somewhere (or maybe not bc pegasus?)

Reply:
This whole announcement for is "en" only, so while it probably is available as a string, it seems a bit simpler to leave it hardcoded.  The other strings are coming direct from DCDO so there won't be any localised strings in this piece of UI.